src/components/Acknowledgment/Acknowledgment.module.scss

```diff
@@ -0,0 +1,59 @@
+@import "../../styles/variables";
+
+.acknowledgment_container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-left: 5px;
+
+  .label {
```

Comment:
Seems weird the labels have more visual weight or even equal visual weight as the names themselves. I think the font could at least be a little smaller.

Reply:
I have reduced the font size.